### PR TITLE
Added version command (plus test)

### PR DESCRIPTION
**Pull request by jaymickey**
Basic `octo.exe version` command. Simply returns the version number.

### CODE REVIEW

File: source/Octo.Tests/Commands/VersionTestFixture.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+            commandOutputProvider = new CommandOutputProvider(logger);
+            versionCommand = new VersionCommand(commandOutputProvider);
+            logger = new LoggerConfiguration().WriteTo.TextWriter(output).CreateLogger();
+        }
+
+        [Test]
+        public void ShouldPrintCorrectVersionNumber()
+        {
+            var version = GetVersionFromFile(Path.Combine(TestContext.CurrentContext.WorkDirectory, "ExpectedSdkVersion.txt"));
```
Comment (mcasperson) [inline source/Octo.Tests/Commands/VersionTestFixture.cs]: ExpectedSdkVersion.txt needs to be added to .gitignore as it is not removed after a test, but should not be included in the repo. If it is not excluded and IDE will incorrectly automatically include it at some point.

**Comment by mcasperson** [comment]
This PR is good to go save for ignoring the test text file that gets created.

**Comment by jaymickey** [comment]
Hi @mcasperson,

Absolutely right, I've added change to .gitignore.

**Comment by mcasperson** [comment]
Thanks for that @jaymickey .

**Pull request merged**